nvflare/apis/resource_manager_spec.py

```diff
@@ -81,7 +74,7 @@ def allocate_resources(self, resource_requirement: dict, token: str, fl_ctx: FLC
 
     @abstractmethod
     def free_resources(self, resources: dict, token: str, fl_ctx: FLContext):
-        """Free the resources.
+        """Frees resources.
```

Comment:
frees ? should be just "free"

Reply:
Why should this be "free"?

I follow this: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
I see. OK